Userland/Libraries/LibWeb/HTML/WorkerGlobalScope.idl

```diff
@@ -1,6 +1,7 @@
 #import <CSS/FontFaceSet.idl>
 #import <DOM/EventTarget.idl>
 #import <DOM/EventHandler.idl>
+#import <HTML/Window.idl>
```

Comment:
Why do we need this import?

Reply:
it's related to https://github.com/LadybirdBrowser/ladybird/issues/1855
it just needs to be imported before `HTML/WindowOrWorkerGlobalScope.idl` just try to remove it and recompile, it fails 😓